lib/main_containers.pm

```diff
@@ -80,7 +80,7 @@ sub load_host_tests_docker {
         loadtest 'containers/registry' if is_x86_64;
         loadtest 'containers/docker_compose';
     }
-    loadtest 'containers/validate_btrfs' if is_x86_64;
+    loadtest 'containers/validate_btrfs' if (is_x86_64 && !is_jeos);
```

Comment:
This module is not executed in SLE JeOS either, so I created this ticket to enable it later on.
https://progress.opensuse.org/issues/102377